yarn.lock

```diff
@@ -1691,7 +1691,7 @@
     y-websocket "^1.3.15"
     yjs "^13.5.17"
 
-"@jupyterlab/docregistry@^3.2.9":
+"@jupyterlab/docregistry@^3.2.9", "@jupyterlab/docregistry@~3.2.9":
```

Comment:
should probably align these

Reply:
I think we have a couple of `@jupyterlab/` dependencies like this.

Maybe we can do one more pass on the `yarn.lock` as a follow-up.